installer/build/container/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM photon:2.0
+
+ENV GOVERSION=1.10.5
+ENV PATH=$PATH:/root/gsutil:/usr/local/go/bin:/usr/local/google-cloud-sdk/bin/
+
+RUN set -eux; \
+    tdnf remove -y toybox; \
+    tdnf install -y make tar gzip python2 python-pip sed git diffutils findutils rpm \
+    gawk util-linux which docker gptfdisk e2fsprogs grub2-efi efibootmgr parted dosfstools xz docker; \
+    curl -L'#' -k https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-200.0.0-linux-x86_64.tar.gz  | tar xzf - -C /usr/local; \
+    mkdir -p /root/.gsutil/; \
+    /usr/local/google-cloud-sdk/install.sh --quiet; \
```

Comment:
Why do we need google-cloud-sdk?  It's quite large and takes awhile to install.

Reply:
removed